website/docs/d/schema_site_bd_l3out.html.markdown

```diff
@@ -14,22 +14,22 @@ Data source for MSO Schema Site Bridge Domain L3out.
 
 ```hcl
 
-data "mso_schema_site_bd_l3out" "bdL3out" {
-  schema_id  = data.mso_schema.schema1.id
-  site_id    = data.mso_site.site1.id
-  bd_name    = "WebServer-Finance"
-  l3out_name = "ccc"
+data "mso_schema_site_bd_l3out" "example" {
+  schema_id     = data.mso_schema.schema1.id
+  site_id       = data.mso_site.site1.id
+  template_name = "Template1"
+  bd_name       = "WebServer-Finance"
+  l3out_name    = "ccc"
 }
 
 ```
 
 ## Argument Reference ##
 
-* `schema_id` - (Required) SchemaID under which you want to deploy Bd L3out.
-* `site_id` - (Required) SiteID under which you want to deploy Bd L3out.
-* `bd_name` - (Required) Name of Bridge Domain.
-* `l3out_name` - (Required) Name of L3out to manage.
-
-## Attribute Reference ##
-
-* `template_name` - (Optional) Template where Bd L3out to be created.
+* `schema_id` - (Required) The schema ID under which the BD is deployed.
+* `site_id` - (Required) The site ID under which the BD is deployed.
+* `template_name` - (Required) The template name under which the BD is deployed.
+* `bd_name` - (Required)  The name of the BD.
+* `l3out_name` - (Required) The name of the L3out.
+* `l3out_schema_id` - (Optional) The schema ID of the L3out. The `schema_id` of the BD will be used if not provided. 
+* `l3out_template_name` - (Optional) The template name of the L3out. The `template_name` of the BD will be used if not provided. 
```

Comment:
should it be Optional -> Read-only ?

Reply:
no since this is possible input on datasource